roles/calico/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+calico_image: ciscocloud/calico-node
+calico_image_tag: v0.4.9-mi
+
+bintray_baseurl: "https://bintray.com/artifact/download/loshkovskyi/generic"
```

Comment:
Still waiting for access to upload the files. Will ask if there are any updates.